container/crio/handler.go

```diff
@@ -225,6 +225,9 @@ func (self *crioContainerHandler) needNet() bool {
 func (self *crioContainerHandler) GetSpec() (info.ContainerSpec, error) {
 	hasFilesystem := self.includedMetrics.Has(container.DiskUsageMetrics)
 	spec, err := common.GetSpec(self.cgroupPaths, self.machineInfoFactory, self.needNet(), hasFilesystem)
+	if err != nil {
+		return info.ContainerSpec{}, err
```

Comment:
I think we still want to set the Lables, Envs, and Image here... I don't think this is needed.